card/mod/01_history/set/all/history.rb

```diff
@@ -333,3 +333,7 @@ def diff_args
 end
 
 
+def has_edits?
+  Card::Act.where(:actor_id=>id).where('card_id IS NOT NULL').present?
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.